README.md

```diff
@@ -53,31 +53,53 @@ Install NPM package
     import {HDKEY, DID} from '@functionland/fula-sec'
 
 
-    // Prefix moc keys
-    let secretKey = '123456789'
-    let signedKey = '9d7020006cf0696334ead54fffb859a8253e5a44860c211d23c7b6bf842d0c63535a5efd266a647cabdc4392df9a4ce28db7dc393318068d93bf33a32adb81ae';
+    /* Prefix moc keys */
+    let password = '123456789'  //User`s password
+    let signedKey = '9d7020006cf0696334ead54fffb859a8253e5a44860c211d23c7b6bf842d0c63535a5efd266a647cabdc4392df9a4ce28db7dc393318068d93bf33a32adb81ae'; // signedKey from metamask
 
     
-    /* Add user`s secretKey */
-    const ed = new HDKEY(secretKey);
+    /* 1 - Add user`s password */
+    const ed = new HDKEY(password);
     
-    // 1. Sign with chaincode   |chainCode| --->  |Metamask|      
-    // 2. Get signedKey         |signedKey| <---  |Metamask| 
+    // A. Sign with chaincode   |chainCode| --->  |Metamask|      
+    // B. Get signedKey         |signedKey| <---  |Metamask| 
 
-    /* Get chainCode to get signedKey from Metamask*/
+    /* 2 - Get chainCode to get signedKey from Metamask*/
     const chainCode = ed.chainCode;
+    `type:base64pad APSWnk8ULP/v//oseMeSEDadMBSSeX/SOxOREYhjQ7g=`
     /* Send request to metamask*/
```

Comment:
can you please change to:
/* chainCode is created from part of the password to be sent to the signing authority like MetaMask wallet to get a unique signature back*/